app/views/admin/cms/show.html.erb

```diff
@@ -0,0 +1,23 @@
+<% content_for :cms_heading do %>
```

Comment:
Okay, so yeah, this is kind of a strange way of doing things. A more standard approach would be:

1) This file keeping the `content_for :cms_heading ...` bit as it is.

2) Remove the `content_for :cms_collection_list` block around the rest of this view, and just render it into the view normally.

3) In the `cms` layout: Remove the `content_for` rendering of `:cms_collection_list` and keep just the normal `yield`.